mongo/CHANGELOG.md

```diff
@@ -1,75 +1,51 @@
 # CHANGELOG - mongo
 
-1.5.4 / Unreleased
-==================
-### Changes
+## 1.5.4 / Unreleased
 
 * [IMPROVEMENT] Allow disabling of replica access. See #1516
 
-1.5.3 / Unreleased
-==================
-### Changes
+## 1.5.3 / 2018-05-11
 
 * [BUGFIX] Added `top` metrics ending in `countsps` that properly submit as `rate`s. See #1491
```

Comment:
This is my bad. Should be `countps` https://github.com/DataDog/integrations-core/pull/1498/files#diff-a8f6fbe6977246779aba799e2c973cd2